mantidproject/mantid
# Adding beam stop arm masking for all ISIS SANS instruments

The beam stop arm masking in the new SANS Gui was only enabled for SANS2D. This enables it for LOQ, LARMOR and ZOOM as well.

**To test:**

- Get the data from the following folder
[loqdemo.zip](https://github.com/mantidproject/mantid/files/1895126/loqdemo.zip)

- Open up the SANS GUI V2. Load MaskFile.txt as the user file and batch.csv as the batch file.
- Go to settings->Masks and click display mask confirm that a beam stop arm is masked out.



Fixes #22266 . 

**Release Notes** 

Have updated release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
